src/services/cache-config/CacheConfigService.ts

```diff
@@ -345,12 +346,16 @@ export class CacheConfigService {
     return deferred.promise;
   }
 
-  private createHelper(builder: CompositeOpBuilder, address: string[], config: any): void {
+  private createHelper(builder: CompositeOpBuilder, address: string[], config: any, requiredNode?: boolean): void {
```

Comment:
@ryanemerson I changed the createHelper function slightly to include optional parameter requiredNode. If true, we'll add required-node field to a config object. See below